src/main/resources/application.yml

```diff
@@ -42,6 +44,8 @@ spring:
       hibernate:
         dialect: org.hibernate.dialect.PostgreSQLDialect
         format_sql: true
+        jdbc:
+          batch_size: 50
```

Comment:
This part is what actually enables batched inserts. I picked 50 because that's the max value recommended in the hibernate docs